lib/features/modeling/behavior/EventBasedGatewayBehavior.js

```diff
@@ -0,0 +1,34 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { is } from '../../../util/ModelUtil';
+
+export default function EventBasedGatewayBehavior(eventBus, modeling) {
+
+  CommandInterceptor.call(this, eventBus);
+
+  // remove existing connection before connecting from event-based gateway
+  this.preExecuted('connection.create', function(event) {
+
+    var source = event.context.source,
+        target = event.context.target,
+        oldConnections = target.incoming;
+
+    if (
+      is(source, 'bpmn:EventBasedGateway') &&
+      target.incoming.length > 0
```

Comment:
This handles multiple connections, despite the [test](https://github.com/bpmn-io/bpmn-js/pull/1006/commits/eb4aa6f354888598c4fdae72a68c1b4ef99c8b31#diff-c08486b559e39e159cf3632d833d3fc9R19) checking only for single connections.

This is because after ddc5c02, multiple connections are disallowed. However, it may be that a diagram is imported which was created with a tool that does not disallow multiple connections, so better to cover the case with multiple connections.